region/lowernorfair/west/Golden Torizo's Room.json

```diff
@@ -607,6 +616,11 @@
         {"heatFrames": 200}
       ]
     },
+    {
+      "link": [2, 2],
+      "name": "Crystal Flash",
+      "requires": ["h_canHeatedCrystalFlash"]
```

Comment:
It doesn't look like you can avoid a GT hit if he's alive.

Reply:
It seems to work if you CF immediately on entry and then morph and roll under. Added a note